README.md

```diff
@@ -1,56 +1,26 @@
 ##  little-vm-helper
 
-little-vm-helper is a VM management toolset, targeting testing and development, especially with
-features that are dependent the kernel, such as BPF. It should not be used for running production
-VMs. That is, it is only intended for VMs that have a "fire-once" lifetime. Booting fast, building
-images fast, and being storage efficient are the main goals.
+little-vm-helper (lvh) is a VM management tool, aimed for testing and development of features that
+depend on the kernel, such as BPF. It is used in [cilium](https://github.com/cilium/cilium) and
+[tetragon](https://github.com/cilium/teragon). It can also be used for kernel development. It is not
```

Comment:
Also, pwru started using it - https://github.com/cilium/pwru/pull/111.

Reply:
Added!